test/ejabberd_tests/tests/amp_SUITE.erl

```diff
@@ -155,12 +155,12 @@ actions_and_conditions_discovery_test(Config) ->
               escalus_client:send(Alice, disco_info_amp_node(Config)),
               Response = escalus_client:wait_for_stanza(Alice),
               assert_has_features(Response,
-                                  [ns_amp()
-                                  ,<<"http://jabber.org/protocol/amp?action=notify">>
-                                  ,<<"http://jabber.org/protocol/amp?action=error">>
-                                  ,<<"http://jabber.org/protocol/amp?condition=deliver">>
-                                  ,<<"http://jabber.org/protocol/amp?condition=match-resource">>
-                                  ])
+                [ns_amp()
```

Comment:
The list should be indented more so it's aligned with the beginning of `Response` var in the above line. I understand that Elvis is complaining about too long lines here. If so I suggest assigning the list to a var and using this var in `assert_has_features`.